pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/ManagedCursorMetricsTest.java

```diff
@@ -37,13 +37,13 @@
 @Test(groups = "quarantine")
```

Comment:
if the test is no more flaky, please replace "quarantive" with "broker" group to make it run along with the other tests.

Reply:
Ok, thanks.